civihr_employee_portal/src/View/MyDetails_MyRoleView.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\View;
+
+class MyDetails_MyRoleView extends AbstractView {
+
+  /**
+   * @var string
+   */
+  protected static $name = 'MyDetails_MyRole';
+
+  /**
+   * @inheritdoc
+   */
+  public function alter($view, $query) {
```

Comment:
Please document(in the doc section of the code) what is happening here.

Reply:
Of course, I overlooked that.